app.css

```diff
@@ -0,0 +1,33 @@
+body {
+    background: url(https://images.unsplash.com/photo-1415369629372-26f2fe60c467);
+    background-size: cover;
+    background-position: center; 
+}
+
+body,
+html {
+    width: 100%;
+    height: 100%;
+    font-family: "Lato";
+    color: white;
+}
+
+h1 {
+	font-weight: 700;
+	font-size: 5em;
+}
+
+
+.content{
+	padding-top: 25%;
```

Comment:
Try to follow a code style, above you have extra space on the left, at the bottom you have less. 
try to use 2 spaces for indentation.